nvgpu: class_create() lost its owner argument in 6.4

diff --git a/drivers/gpu/nvgpu/os/linux/ioctl.c b/drivers/gpu/nvgpu/os/linux/ioctl.c
index 9c25a55..19f6d32 100644
--- a/drivers/gpu/nvgpu/os/linux/ioctl.c
+++ b/drivers/gpu/nvgpu/os/linux/ioctl.c
@@ -430,7 +430,7 @@ static struct nvgpu_class *nvgpu_create_class(struct gk20a *g, const char *class
 #if defined(NV_CLASS_CREATE_HAS_NO_OWNER_ARG) /* Linux v6.4 */
 	class->class = class_create(class_name);
 #else
-	class->class = class_create(THIS_MODULE, class_name);
+	class->class = class_create(class_name);
 #endif
 	if (IS_ERR(class->class)) {
 		nvgpu_err(g, "failed to create class");
